tls_assets.go

```diff
@@ -4,10 +4,18 @@ package certificatetpr
 // These are used when getting a secret from the k8s API, to identify the component the secret belongs to
 type ClusterComponent string
 
+func (c ClusterComponent) String() string {
+	return fmt.Sprintf("%s", string(c))
```

Comment:
Good point. We don't want to change the content though, just return a different representation.